parastell/invessel_build.py

```diff
@@ -18,6 +20,38 @@
 )
 
 
+def create_moab_tris_from_verts(corners, mbc, reverse=False):
+    """Create 2 moab triangle elements from a list of 4 pymoab verts.
+
+    Arguments:
+        corners (4x3 numpy array): list of 4 (x,y,z) points. Connecting the
+            points in the order given should result in a polygon
+        mbc (pymoab core): pymoab core instance to create elements with.
+
+    Returns:
+        list of two pymoab MBTRI elements
+    """
+    if reverse:
+        tri_1 = mbc.create_element(
+            types.MBTRI, [corners[0], corners[1], corners[2]]
+        )
+        tri_2 = mbc.create_element(
+            types.MBTRI, [corners[0], corners[2], corners[3]]
+        )
+    else:
+        tri_1 = mbc.create_element(
+            types.MBTRI, [corners[2], corners[1], corners[0]]
+        )
+        tri_2 = mbc.create_element(
+            types.MBTRI, [corners[3], corners[2], corners[0]]
+        )
+
+    return [tri_1, tri_2]
+
+
+export_allowed_kwargs = ["export_cad_to_dagmc", "dagmc_filename"]
```

Comment:
This can be removed since IVB CAD-to-DAGMC functionality was removed in a previous PR. I'm guessing this made its way back in during your rebase.